package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "webpack -wd --config webpack.example.config.js",
     "prepublish": "NODE_ENV=production webpack -p",
-    "test": "karma start",
+    "test": "NODE_ENV=test karma start",
```

Comment:
This suppresses annoying warnings in CSSTransitionGroup. Long term, I want to disable css transitions (how do I do this?)